gwsumm/__main__.py

```diff
@@ -854,8 +854,8 @@ def main(args=None):
                 issues=issues, writedata=not args.html_only,
                 writehtml=not args.no_html)
 
-        # archive this tab
-        if args.archive:
+        # archive this tab, skip home tab
+        if args.archive and name != "_Home":
```

Comment:
Since the tab name of `_Home` is this in `defaults.ini`, does it matter that gwsumm is saving anything?
```
[tab-home]
type = link
name = _Home
url = /~%(user)s/summary
```
Maybe my question is along the lines of what is the exclusion of tabs named `_Home` saving us?

Reply:
Ok, but it looks like it takes less than 1 second...?

Reply:
It's the time between the second and third lines, so approximately 2 minutes.

Reply:
@iaraota This seems to be saying when it started and when it finished the archiving for the previous tab

Reply:
The "<tab> complete" message happens after the save. If you check the message before already have a saving statement:

```
Writing HTML:
    day/20230823/sei/ham1/isolated.html written
    day/20230823/sei/ham1/ready.html written
gwsumm 2023-08-24 08:08:09 CDT     INFO: Writing data to archive
gwsumm 2023-08-24 08:10:49 CDT    DEBUG: Archive written to /home/detchar/public_html/summary/day/20230823/archive/L1-L1SEI_HAM-1376784018-86400.h5
gwsumm 2023-08-24 08:10:49 CDT    DEBUG: SEI/_HAM1 complete
gwsumm 2023-08-24 08:10:49 CDT     INFO: Writing data to archive
gwsumm 2023-08-24 08:13:27 CDT    DEBUG: Archive written to /home/detchar/public_html/summary/day/20230823/archive/L1-L1SEI_HAM-1376784018-86400.h5
gwsumm 2023-08-24 08:13:27 CDT    DEBUG: _Home complete
gwsumm 2023-08-24 08:13:27 CDT     INFO: -- Data products written, all done --
```

Reply:
Ok, thanks for clarifying! I agree this seems redundant. Do you know what is actually happening when `_Home` is saved?

Reply:
I believe it is saving the same data is saved when the last tab completed, as it saves the `globalv`, which should not change when processing the home tab.

Reply:
@eagoetz  I will actually remove this change from here. I think we could add a new configuration key, something like `archive-data = true/false`. That way the change would be more general and not specific on our configurations. I think we can also use this in tabs that creates no new plots, like the summary tab.